democracy_club/templates/about/about_base.html

```diff
@@ -3,7 +3,8 @@
     <nav class="ds-subnav" aria-label="true">
         <ul>
             <li>
-                <a href="{% url "about" %}">About</a>
+                <a href="{% url "about" %}#funding"
+                >About</a>
```

Comment:
> A few inline comments, but I'm mainly confused by the funding / about pages at this stage. The table is on both about and funding so that at least should be changed. I think the intent is to have all the funding stuff on the same page (not the about home), and then to change the about link to remove the #funding fragment.

Looking back at Mevan's comment, I misunderstood what she wrote: "On About page, the about submenu link “funding”  links to “How is Democracy Club funded?” on the About page." Do you still want it this way?

Reply:
With funding subnav linking to that section?

Reply:
To be clear, suggested sub nav: 

          ```
  <li>
                <a href="{% url "about" %}">About</a>
            </li>
            <li>
                <a href="{% url "about" %}#funding">Funding</a>
            </li>
            <li>
                <a href="{% url "team" %}">The Team</a>
            </li>
            <li>
                <a href="{% url "jobs" %}">Jobs</a>
            </li>
```

Reply:
With this suggestion, I'm not sure what's to become of the current funding page https://democracyclub.org.uk/about/support_us/ as we've extracted the policy info
See here for reference https://docs.google.com/document/d/1O9-NwA0vyFthDn4GniTC13zDQV-94lDa_UF_6f9LrJE/edit#

Reply:
I think for this PR, `/support-us` and `about/support_us/` gets a temporary redirect to `/about` and then I can update https://github.com/DemocracyClub/Website/pull/595 so that `/support-us` and `about/support_us/` goes to `/donate`. What do you think?

Reply:
Decision: Move all funding links and content to a dedicated funding page at `/about/funding`; any donation related links will be handlded in https://github.com/DemocracyClub/Website/pull/595